src/dev/src/app/datagrid/binding-properties/binding-properties.html

```diff
@@ -16,20 +16,27 @@ <h2>Binding model properties to columns</h2>
 </p>
 <p>
   You can also see in the following example how every feature we offer is always opt-in: we did not declare any binding on
-  the "User ID" column, which means it is not sortable of filterable.
+  the "Favorite Color" column, which means it is not sortable of filterable.
+</p>
+
+<h4>Column types</h4>
+<p>
+  By default, bound columns are assumed to contain string-like contents and the user is presented with the normal string filter.
+  If you know that the contents of the column will be numeric, you can instead use the built-in numeric range filter by adding
+  <code class="clr-code">clrDgColType=&quot;'number'&quot;</code>.  You can see an example of this in the ID column.
 </p>
 
 <clr-datagrid>
-  <clr-dg-column>User ID</clr-dg-column>
-  <clr-dg-column [clrDgField]="'name'" [(clrFilterValue)]="nameFilter">Name</clr-dg-column>
-  <clr-dg-column [clrDgField]="'creation'">Creation date</clr-dg-column>
-  <clr-dg-column [clrDgField]="'pokemon.name'">Pokemon</clr-dg-column>
-  <clr-dg-column [clrDgField]="'color'">Favorite color</clr-dg-column>
+  <clr-dg-column clrDgField="creation">Creation date</clr-dg-column>
+  <clr-dg-column clrDgField="id" clrDgColType="number">User ID</clr-dg-column>
+  <clr-dg-column clrDgField="name" [(clrFilterValue)]="nameFilter">Name</clr-dg-column>
+  <clr-dg-column clrDgField="pokemon.name">Pokemon</clr-dg-column>
+  <clr-dg-column>Favorite color</clr-dg-column>
 
   <clr-dg-row *clrDgItems="let user of users">
+    <clr-dg-cell>{{user.creation | date}}</clr-dg-cell>
     <clr-dg-cell>{{user.id}}</clr-dg-cell>
     <clr-dg-cell>{{user.name}}</clr-dg-cell>
-    <clr-dg-cell>{{user.creation | date}}</clr-dg-cell>
```

Comment:
Can you put the `creation` column back in the same place?

Reply:
I rearranged it because otherwise we run into this bug (https://github.com/vmware/clarity/issues/1262#issuecomment-420265815 and https://github.com/vmware/clarity/issues/2683)

![image](https://user-images.githubusercontent.com/7132414/57271755-923c7200-7045-11e9-9ec3-19bc4b36addb.png)

It looks like this may take a while to get resolved -- should I add one more additional column in the first position or what?

Reply:
Oh yes, that is something that is being actively addressed. I see why you did it now, and you can leave it.